packages/mgt-components/src/components/mgt-tasks/mgt-tasks.scss

```diff
@@ -7,4 +7,226 @@
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+@import '../../styles/shared-styles.scss';
+@import '../../../../../node_modules/office-ui-fabric-core/dist/sass/References';
+
+$task-icon-alignment: var(--task-icon-alignment, flex-start);
+$task-icon-background: var(--task-icon-background, transparent);
+$task-icon-background-completed: var(--task-icon-background-completed, $ms-color-sharedGreenCyan10);
+$task-icon-border: var(--task-icon-border, solid 1px $ms-color-gray120);
+$task-icon-border-completed: var(--task-icon-border-completed, solid 1px white);
+$task-icon-border-radius: var(--task-icon-border-radius, 50%);
+$task-icon-color-completed: var(--task-icon-color-completed, white);
+$task-border-completed: var(--task-complete-border, 2px dotted inherit);
+
+:host {
+  background-color: var(--neutral-layer-2);
+  padding: 12px;
+
+  .Header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-bottom: 14px;
+
+    .Title {
+      justify-content: left;
+    }
+
+    .NewTaskButton {
+      justify-content: right;
+      &::part(control) {
+        font-weight: 700;
+      }
+      &::part(start) {
+        margin-top: 6px;
+      }
+    }
+  }
+
+  .Tasks {
+    display: flex;
+    flex-direction: column;
+    row-gap: 20px;
+
+    .Task {
+      background-color: var(--neutral-layer-1);
+      border-radius: 4px;
+      padding: 10px;
+      display: flex;
+      column-gap: 4px;
+      align-items: flex-start;
+      justify-content: space-between;
+
+      &.Complete {
+        border: $task-border-completed;
+      }
+      &.Incomplete {
+        border: 1px dotted var(--neutral-layer-3);
+      }
+
+      .TaskDetailsContainer {
+        display: flex;
+        flex-direction: column;
+        row-gap: 12px;
+        width: 100%;
+
+        .Top {
+          display: flex;
+          justify-content: space-between;
+          column-gap: 4px;
+
+          &.AddNewTask {
+            flex-direction: row;
+          }
+          .CheckAndTitle {
+            display: flex;
+            align-items: flex-start;
+            flex-direction: column;
+            width: 100%;
+            row-gap: 12px;
+
+            .TaskContent {
+              display: grid;
+              grid-auto-flow: column;
+              column-gap: 12px;
+            }
+          }
+
+          .TaskOptions {
+            &.NewTaskActionButtons {
+              display: flex;
+              flex-direction: column;
+              row-gap: 12px;
+            }
+          }
+        }
+        .Bottom {
+          display: grid;
+          grid-auto-flow: column;
+          grid-auto-columns: 25%;
+
+          .TaskGroup,
+          .TaskBucket,
+          .TaskDue {
+            display: flex;
+            align-items: center;
+            gap: 6px;
+
+            .TaskIcon {
```

Comment:
The icons we have here are different sizes and don't align quite right
![image](https://user-images.githubusercontent.com/7122716/228269828-f192a1e1-61f0-4d2a-b90a-316970889746.png)

Can we adjust this to better align these icons?

Reply:
I've used the planner icon and it aligns well. Let me find a better calendar icon 💪

Reply:
I realized that the two icons are built differently. The top one has a _padding_ around it which centers it (about 1px). The bottom one doesn't (it's filling its space point to point). So, when you adjust the widths, unless it's individually done, they remain the same. I prefer having the icons adjusted together as it allows more extensibility.